src/main/java/com/glucoseguardian/webbackend/storage/dto/AssunzioneFarmacoDto.java

```diff
@@ -10,17 +10,17 @@
 @JsonInclude(Include.NON_ABSENT)
 public class AssunzioneFarmacoDto implements Serializable {
 
-  private long id;
-  private long idFarmaco;
-  private int dosaggio;
+  private Long id;
```

Comment:
Isn't it better to also use wrapper classes around primitives in the daos? In order to prevent a lot of autoboxing and unboxings?

Reply:
(same for all the other dto)